charts/thoras/values.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 
-thorasVersion: "2.0.10"
+thorasVersion: "2.0.11"
```

Comment:
Do we always bump the version? Or should this only when we're ready to cut a new release?

Reply:
This is when we cut a new release in the app!

Reply:
This is where we vbump the helm chart:
https://github.com/thoras-ai/helm-charts/blob/d7476ed9c041b873f70c07c7f9782b79af97faf2/charts/thoras/Chart.yaml#L18